packages/headless/src/modules/super-admin/service/index.ts

```diff
@@ -0,0 +1,55 @@
+import { HttpStatus, Injectable } from "@nestjs/common";
+import { PartialType } from "@nestjs/swagger";
+import { SuperAdmin } from "src/entity/super-admin";
+import { errorResponse, successResponse } from "src/utils/response";
+import { IServiceResponse } from "src/utils/response/service.response.interface";
+import { SuperAdminRepository } from "../repositories";
+import { SuperAdminLoginDto, SuperAdminLoginRes } from "../rest/dto/login.dto";
+import { SuperAdminSignupReq } from "../rest/dto/signup.dto";
+import * as bcrypt from 'bcrypt'; 
+import { authConfig } from "config/auth";
+import { JwtService } from "@nestjs/jwt";
+import { AdminJwtPayload } from "src/entity/auth";
+
+@Injectable()
+export class SuperAdminService{
+    constructor(private readonly superAdminRepository: SuperAdminRepository, private readonly jwtService: JwtService){}
+
+    async superAdminCreate(body: SuperAdminSignupReq): Promise<IServiceResponse<Partial<SuperAdmin>>>{ 
+        const isExist = await this.superAdminRepository.findOne({ $or: [ {email: body.email}, {phone: body.phone} ] })
```

Comment:
super admin should use only email & password for login
phone number is an extra info just

Reply:
login works only with email and password not phone number.

Reply:
can we create two account using different email and without phone number?

Reply:
yes